ideascube/search/migrations/0002_reindex.py

```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations, connections
-from ideascube.search.utils import reindex_content
-
-
-def update_reindex(*args):
-    cursor_default = connections['default'].cursor()
-    cursor_default.execute("DROP TABLE IF EXISTS idx")
```

Comment:
Yes, the `0001_initial` does create the table in the transient db now.

This is all a bit of a mess, though.

Ideally we'd just want to completely remove all migrations from the search app, because the model is unmanaged, and as such we actually don't want to use migrations for it.